scripts/devcloud-test.sh

```diff
@@ -0,0 +1,38 @@
+#! /bin/bash
+# Copyright (C) 2023 Intel Corporation
+
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+# See LICENSE.TXT
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+workspace=$1
+compiler_c=$2
+compiler_cxx=$3
+build_type=$4
+
+set -e
+echo "Hostname: $(hostname)"
+
+export PATH=${workspace}/dpcpp_compiler/bin:$PATH
+export CPATH=${workspace}/dpcpp_compiler/include:$CPATH
+
+#These variables is needed only for cmake
+LIBRARY_PATH=${workspace}/dpcpp_compiler/lib \
+LD_LIBRARY_PATH=${workspace}/dpcpp_compiler/lib \
```

Comment:
Thought that these env vars would be required in the building step instead, however if it works then it's ok.

Reply:
It works.